src/components/MyAccount/Settings/SettingsInputForm.tsx

```diff
@@ -42,9 +43,26 @@ const SettingsInputForm = ({
 
   const [tempInputs, setTempInputs] = useState([...inputs])
 
+  const inputRefs = useRef<Array<TextInputRefType | null>>([])
+  const editingRef = useRef<HTMLButtonElement | null>()
+
+  const focusLastInput = () => {
+    const lastIndex = tempInputs.length - 1
+    if (inputRefs.current[lastIndex]) {
+      inputRefs.current[lastIndex]?.focus()
+    }
+  }
+
+  useEffect(() => {
```

Comment:
With three text inputs open for the phone settings, this focus is triggering on every keystroke input. ie Typing a single character into input 2 jumps the focus to input 3. I think you want an empty array to only run on the first render. I think this will make typescript unhappy but that's ok.

Reply:
updated it to `tempInputs.length` which I think is right because then it updates only when a new input is added/removed